src/CompanyService/Controllers/PositionController.cs

```diff
@@ -33,7 +33,8 @@ public class PositionController : ControllerBase
         }
 
         [HttpGet("find")]
-        public List<PositionResponse> Find([FromServices] IFindPositionsCommand command)
+        public List<PositionResponse> Find(
+            [FromServices] IFindPositionsCommand command)
```

Comment:
Now don't you want to create bool flag 'IncludeDisactivated'?

Reply:
done